Themes/default/css/admin.css

```diff
@@ -624,6 +624,8 @@ h3.grid_header {
 #error_log span.error_message {
 	overflow: hidden; /* This changes how the text wraps around the float */
 	display: block;
+	white-space: pre-wrap;
+	font: 86%/150% "Courier", "Inconsolata", "Apercu Mono", "Nitti", "Monaco";
```

Comment:
Instead of manually defining the font here, I suggest adding `span.error_message` to this list:
https://github.com/SimpleMachines/SMF2.1/blob/888b2677c8bf640776f56d446101c42a4ca4492b/Themes/default/css/index.css#L78

Note: you will first need to merge recent changes to release-2.1 into this PR in order to do that.

Reply:
the white-space wraper needs to stay ther?